exo-deploy/terraform/aws/vars.tf

```diff
@@ -0,0 +1,94 @@
+/* Variables */
+
+variable "ecs_ebs_optimized" {
+  description = "Boolean indicating if ECS instances are EBS-optimized"
+  default     = false
+}
+
+variable "ecs_instance_type" {
+  description = "Instance type to use for ECS instances"
+  default     = "t2.micro"
+}
+
+variable "env" {
+  description = "Name of the environment, used for naming and prefixing"
+}
+
+variable "key_name" {
```

Comment:
You can do it a few different ways:
- Bake an AMI with the correct keys
- Create a temporary key for setup then have something like ansible come in & setup the keys
- Put the keys in an S3 bucket and have a script on the bastion poll periodically